### PR TITLE
Exhibition info box

### PR DESCRIPTION
**Pull request by gestchild**
References #3009

Adds the InfoBox and relevant information to the exhibition base page:

screen shot 2018-09-07 at 11 38 31

Adds the InfoBox Component to Cardigan:

cardigan

~@GarethOrmerod the link hover colour is not accessible on the yellow background, any thoughts?~ -captured in #3303

@Heesoomoon We'll need to fill out the README in storybook, I've added a very basic purpose for now.

- [x] ~Next up: control the inclusion of the family pack from Prismic~ - done

### CODE REVIEW

File: cardigan/stories/components/InfoBox.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import { storiesOf } from '@storybook/react';
+import InfoBox from '../../../common/views/components/InfoBox/InfoBox';
+import { boolean, text } from '@storybook/addon-knobs/react';
+import Readme from '../../../common/views/components/InfoBox/README.md';
+import {font} from '../../../common/utils/classnames';
+
+const InfoBoxExample = () => {
+  const title = text('Title', 'Visit us');
+  const hasMoreInfoLink = boolean('Has more info. link', true);
```
Comment (jamesgorrie) [inline cardigan/stories/components/InfoBox.js]: Should capitalise `link`.

File: common/services/prismic/exhibitions.js [hunk 2]
```diff
@@ -30,10 +31,27 @@ import {parseInstallationDoc} from './installations';
...
+    .map(label => parseResourceType(label.data));
+}
+
+function parseResourceType(fragment: PrismicFragment): Resource {
+  return {
+    title: asText(fragment.title),
+    description: fragment.description,
+    icon: fragment.icon
+  };
+}
```
Comment (jamesgorrie) [inline common/services/prismic/exhibitions.js]: To be really bikesheddy, as these aren't used more than once, nor tested, they could just be in the parse function...

File: common/services/prismic/fetch-links.js [hunk 3]
```diff
@@ -51,3 +51,8 @@ export const eventPoliciesFields = [
   'event-policies.title',
   'event-policies.description'
 ];
+export const exhibitionResourcesFields = [
+  'exhibition-resources.title',
+  'exhibition-resources.description',
+  'exhibition-resources.icon'
+];
```
Comment (jamesgorrie) [inline common/services/prismic/fetch-links.js]: [GraphQL soon](https://www.youtube.com/watch?v=W-QWGw7Zb1c)

File: common/views/components/InfoBox/InfoBox.js [hunk 4]
```diff
@@ -20,43 +20,42 @@ const InfoBox = ({
...
-          <div className={classNames([
-            'plain-text',
-            font({s: 'HNL4'}),
-            spacing({s: 4}, {margin: ['bottom']})
-          ])}>
-            <PrismicHtmlBlock html={description} />
+    <div className={classNames([
+      'bg-yellow',
+      spacing({s: 4}, {padding: ['top', 'right', 'bottom', 'left']}),
+      spacing({s: 4}, {margin: ['bottom']})
```
Comment (jamesgorrie) [inline common/views/components/InfoBox/InfoBox.js]: @davidpmccormick what do you think?

I allowed `classNames` to take arrays OR objects for backwards compatibility, but thought we would have 1 way to rule them all (objects)?

File: common/services/prismic/exhibitions.js [hunk 5]
```diff
@@ -30,10 +31,29 @@ import {parseInstallationDoc} from './installations';
...
 const startField = 'my.exhibitions.start';
 const endField = 'my.exhibitions.end';
 
+function parseResourceTypeList(fragment: PrismicFragment[], labelKey: string): Resource[] {
+  return fragment
+    .map(label => label[labelKey])
+    .filter(Boolean)
+    .filter(label => label.isBroken === false)
+    .map(label => {
+      return parseResourceType(label.data);
```
Comment (davidpmccormick) [inline common/services/prismic/exhibitions.js]: prefer one liner :trollface:

**Pull request merged**